### PR TITLE
Adds avatar to speakers and improves scroll

### LINKED ISSUE #1088: Buggy scroll bar at Featured Speakers section
- The scrollbar disappears instantly with just 2px away from it which result in a buggy situation where a user can not scroll down.

### PR DESCRIPTION
**Pull request by geekyd**
Fixes #1088 and #1129 

Adds avatar for sessions and fixes buggy scroll for morzilla.

Test server:
https://open-event-web-appp.herokuapp.com

@mariobehling @aayusharora Please review. Thanks
Screenshots
**Before**
screen shot 2017-03-06 at 4 21 37 am

**After**
screen shot 2017-03-06 at 4 12 43 am

### CODE REVIEW

File: src/backend/_scss/application.scss [hunk 1]
```diff
@@ -1072,13 +1072,9 @@ a {
   .speaker-description {
     float: left;
     max-height: 262.5px;
-    overflow: hidden;
+    overflow: auto;
```
Comment (aayusharora) [inline src/backend/_scss/application.scss]: @geekyd Overflow : auto will introduce a `scrollbar` which we don't want initially. We want scrollbar only on hover.

Comment (geekyd) [inline src/backend/_scss/application.scss]: @aayusharora I will make it auto on hover for speaker-description

**Comment by geekyd** [comment]
@aayusharora I have made the changes please review. thanks :)

**Comment by aayusharora** [comment]
Thank you @geekyd

**Pull request merged**